tests/components/mass_wasting_runout/test_mass_wasting_runout.py

```diff
@@ -3,6 +3,15 @@
 
 from landlab.components import FlowDirectorMFD
 from landlab.components.mass_wasting_runout import MassWastingRunout
+from landlab.components.mass_wasting_runout.mass_wasting_runout import erosion_coef_k
+from landlab.components.mass_wasting_runout.mass_wasting_runout import erosion_rate
+from landlab.components.mass_wasting_runout.mass_wasting_runout import flow_velocity
+from landlab.components.mass_wasting_runout.mass_wasting_runout import (
+    shear_stress_grains,
+)
+from landlab.components.mass_wasting_runout.mass_wasting_runout import (
+    shear_stress_static,
+)
 
 
 class Test__virtual_laboratory_smoke_tests:
```

Comment:
Please change "Test__virtual" as "Test_virtual". Also, it would be great to define all the test class names to follow the camel case style (e.g., "TestVirtualLaboratorySmokeTests" ) to follow the standard Python class naming conventions (Just a suggestion).

Reply:
Thanks, I changed all of the test class names to camel case style